pom.xml

```diff
@@ -3220,9 +3217,33 @@
 
     <profile>
       <id>scala-2.12</id>
+      <activation>
```

Comment:
> What happens if neither is specified?

The extra `args` defined in `scala-2.12` profile will not appended If not activated by default.

Reply:
> Generally the "2.12" configuration goes in the main POM and then is overridden for non-default profiles here. So if 2.13 needs different config, that's fine, but 2.12's config should be up top. If the problem is overwriting Maven config in a profile, I can show you how to do that.

I'll try to solve it first

Reply:
What I mean is: you put 2.12's config outside a profile. 2.13's config needs to replace it, and you can set this in the Maven XML a certain way to prevent it from just appending, instead making it replace. Otherwise these args won't be there at all for a 2.12 build where any profile is activated, without activating the 2.12 profile explicitly.

Reply:
Address 4fe256 changed a way to achieve the purpose, but still unsatisfactory  :(